src/components/main-view/main-view.jsx

```diff
@@ -1,209 +1,29 @@
-      "Birthdate": 1957
-    },
-    "Featured": true,
-    "Title": "Toy Story"
-  },
-  {
-    "_id": {
-      "$oid": "646dcac574a1324822e199f0"
-    },
-    "Title": "Coco",
-    "Genre": [
-      {
-        "Name": "Animation",
-        "Description": "Animated film is a collection of illustrations that are photographed frame-by-frame and then played in a quick succession. Since its inception, animation has had a creative and imaginative tendency. Being able to bring animals and objects to life, this genre has catered towards fairy tales and children's stories."
-      }
-    ],
-    "Description": "Coco is a 2017 American 3D computer-animated fantasy film...",
-    "Director": {
-      "Name": "Lee Unkrich",
-      "Bio": "Lee Unkrich was born on August 8, 1967, in Cleveland, Ohio...",
-      "ImageURL": "https://encrypted-tbn2.gstatic.com/images?q=tbn:ANd9GcQenOp-f1zeoETh_mDGfGkUjoCr0xz3tfYNU5KbK5dV9iEh3Ltm",
-      "Birthdate": 1967
-    },
-    "Featured": true
-  },
-  {
-    "_id": {
-      "$oid": "646dcae274a1324822e199f1"
-    },
-    "Title": "Inception",
-    "Genre": [
-      {
-        "name": "Thriller",
-        "description": "Thriller is a genre that creates intense excitement, suspense, and anticipation. It often involves high stakes, danger, and unexpected plot twists. Thriller films keep the audience on the edge of their seats and typically feature suspenseful storylines and thrilling action sequences."
-      }
-    ],
-    "Description": "Inception is a 2010 science fiction action thriller film...",
-    "Director": {
-      "Name": "Christopher Nolan",
-      "Bio": "updated Bio",
-      "ImageURL": "https://t0.gstatic.com/licensed-image?q=tbn:ANd9GcS9KbgnntjGPbxTJYW_LOd8Ry7OP9fHgErMRp3J3ZQ16klJ512gEfI3hWFjUuWqqLBE",
-      "Birthdate": 1970
-    },
-    "Featured": true
-  },
-  {
-    "_id": {
-      "$oid": "646dcb0574a1324822e199f2"
-    },
-    "Title": "Interstellar",
-    "Genre": [
-      {
-        "name": "Science Fiction",
-        "description": "Science Fiction is a genre that explores imaginative and futuristic concepts. It often involves advanced technology, scientific possibilities, and futuristic societies. Science fiction films often speculate on the impact of science and technology on individuals and society as a whole."
-      }
-    ],
-    "Description": "Interstellar is a 2014 science fiction film set in a future where humanity is facing extinction...",
-    "Director": {
-      "Name": "Christopher Nolan",
-      "Bio": "updated Bio",
-      "ImageURL": "https://t0.gstatic.com/licensed-image?q=tbn:ANd9GcS9KbgnntjGPbxTJYW_LOd8Ry7OP9fHgErMRp3J3ZQ16klJ512gEfI3hWFjUuWqqLBE",
-      "Birthdate": 1970
-    },
-    "Featured": true
-  }]);
+  const [movies, setMovies] = useState([]);
   const [selectedMovie, setSelectedMovie] = useState (null);
+
+  useEffect(() => {
+    fetch(process.env.CONNECTION_URI)
```

Comment:
Hi @chrisdvadashanta ,

This URL value should be something like: `http://myherokuapi.com/movies`. It is not the same URL as the MongoDB URI. I know that this configurable as an environment variable. Ideally, the configurable part should only be the domain. you could setup a variable like `BACKEND_API=http://myherokuapi.com`. And when using this in your file, you would use it as "...fetch(`${process.env. BACKEND_API}/movies`)..."

For now, I wouldn't recommend using `process.env.CONNECTION_URI` , but rather replace it with the actual API URL. If you have  additional time after the this task, then you can do that. And I will be happy to review it too.

Reply:
thank you